src/graphql/helpers.ts

```diff
@@ -283,15 +308,20 @@ export async function fetchSpaces(args) {
   }
 
   const query = `
-    SELECT s.* FROM spaces s
+    SELECT s.*, skins.*, s.id AS id FROM spaces s
+    LEFT JOIN skins ON s.id = skins.id
     WHERE s.deleted = 0 ${queryStr}
-    GROUP BY s.id
     ORDER BY s.${orderBy} ${orderDirection} LIMIT ?, ?
   `;
   params.push(skip, first);
 
   const spaces = await db.queryAsync(query, params);
-  return spaces.map(space => Object.assign(space, formatSpace(space)));
+  return spaces.map(space =>
+    Object.assign(
+      space,
+      formatSpace({ skinSettings: formatSkinSettings(space), ...space })
```

Comment:
not sure if it is good to call formatSkinSettings on every `formatSpace`, you see some alternative?

could we call `formatSkinSettings` inside `formatSpace`? maybe we can add `skinsettings` inside `settings` object 🤔 

But yea not too important